package.json

```
@@ -12,18 +12,31 @@
     "generate": "turbo run generate",
     "typecheck": "turbo run typecheck",
     "version": "changeset version",
-    "release": "changeset tag"
+    "release": "changeset tag",
+    "docker:pregenerate:env": "shx rm -rf docker/generated && shx mkdir docker/generated",
+    "docker:generate:env": "yarn run docker:pregenerate:env && yarn run docker:generate:env:cosmosdb && yarn run docker:generate:env:io-session-manager && yarn run docker:generate:env:io-lollipop",
+    "docker:generate:env:cosmosdb": "dotenv -e docker/.env.common envsub docker/cosmosdb/env-dev docker/generated/.env.cosmosdb",
+    "docker:generate:env:io-session-manager": "dotenv -e docker/.env.common envsub docker/io-session-manager/env-dev docker/generated/.env.io-session-manager",
+    "docker:generate:env:io-lollipop": "dotenv -e docker/.env.common envsub docker/io-lollipop/env-dev docker/generated/.env.io-lollipop",
+    "docker:build": "docker compose build",
+    "docker:start": "docker compose up -d",
+    "docker:start:build": "docker compose up --build -d",
+    "test:integrations": "yarn run test:integrations:io-lollipop",
+    "test:integrations:io-lollipop": "docker exec io-mock-io-lollipop-1 /bin/bash -c 'yarn run vitest run src/__integrations__/__tests__ -c src/__integrations__/vite.config.mts' --no-file-parallelism"
```

```suggestion
    "test:integrations:io-lollipop": "docker compose exec io-lollipop /bin/bash -c 'yarn run vitest run src/__integrations__/__tests__ -c src/__integrations__/vite.config.mts' --no-file-parallelism"
```
please use compose plugin to eliminate the need to add the project prefix and container number suffix